te_markdown/terminology.md

```diff
@@ -8,7 +8,33 @@ Test-Editor-Web is a lightweight application for specifying, implementing, and e
 
 ## Overview
 
-<!--- TODO: insert (annotated?) overview image and briefly describe the main areas of the user interface -->
+![Overview](/images/te.overview.svg?sanitize=true){:class="img-responsive"}
+
+### The different areas of the Test-Editor-Web
+In the screenshot above you can see the different areas of the Test-Editor-Web which are divided
+through the different yellow areas. The description of the specific area can be found under the number of this area underneath.   
+
+#### 1. The Test-Navigator area
+In this area the user can specify the structure of the different test artifacts.
+He can create, delete, rename, cut, copy and paste test artifacts and refresh the workspace.
+The user can filter the different test-artifacts, so that just the specific test-artifact is visible which was selected.
```

Comment:
Also seems a bit too abstract… I think you'll need to introduce the artifact types, at least as examples, here. Or talk about it in terms of the languages being used ("test specification language (TSL)" and "test case language (TCL)" are already used in other places, and are part of our glossary).